src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,97 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 1 << 4;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int size;
+
+    class Node<K, V> {
+        private final int hash;
+        private K key;
+        private V value;
+        private Node<K, V> next;
+
+        Node(int hash, K key, V value, Node<K, V> next) {
+            this.hash = hash;
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
```

Comment:
Let's just remove this parameter from the constructor and set it manually when we need, because in some cases you just pass `null` value
```suggestion
        Node(int hash, K key, V value) {
            this.hash = hash;
            this.key = key;
            this.value = value;
        }
```